tv-app.html

```diff
@@ -0,0 +1,244 @@
+          life tours of ancient Rome.
+        </p>
+      </div>
+      <div class="col-sm-6 phone-img-pad">
+        <img
+          class="right"
+          src="//images.ctfassets.net/y3a9myzsdjan/5PlzGI7r5BksY8pimswEsd/8db0bac5d4c6c0bdbbe04c8c0e17143f/tvapp-kids-club-tile-videos_2x.png"
+          data-optimize-img
+          style="width: auto;"
+        />
+      </div>
+    </div>
+  </div>
+
+  <!-- 4th Page, Prayer -->
+  <div class="container tv-spacing">
+    <div class="row parent">
+      <div class="col-sm-push-6  col-sm-6 content-middle">
+        <h2 class="flush-bottom font-family-condensed-extra text-uppercase">
+          WORSHIP AT HOME
+        </h2>
+        <p class="font-size-base lead">
+          Original songs, worship sets and exclusive behind the scenes footage
+          with Crossroads Music.
+        </p>
+      </div>
+      <div class="col-sm-pull-6  col-sm-6 phone-img-pad">
+        <img
+          class="left"
+          src="//images.ctfassets.net/y3a9myzsdjan/60WiFavMwY7l3ajTrJsocX/80599ef6da327ed1e78aa1319b1a9bf8/tvapp-music-tile_2x.png"
+          data-optimize-img
+          style="width: auto;"
+        />
+      </div>
+    </div>
+  </div>
+
+  <!-- 5th Page, Latest Message -->
+  <div class="container tv-spacing">
+    <div class="row parent">
+      <div class=" col-sm-6 content-middle">
+        <h2 class="flush-bottom font-family-condensed-extra text-uppercase">
+          GET BACK IN THE HABIT
+        </h2>
+        <p class="font-size-base lead">
+          Our schedules have all been hijacked. The Crossroads TV App makes it
+          easy to bring your family together for church.
+        </p>
+      </div>
+      <div class=" col-sm-6 phone-img-pad">
+        <img
+          class="right"
+          src="//images.ctfassets.net/y3a9myzsdjan/4XqUSJrlmI5swaUnkz52e3/7aa72ecb0c0cbf8a84e9fdac530d0e36/tvapp-temp-family-tv_2x.png"
+          data-optimize-img
+          style="width: auto;"
+        />
+      </div>
+    </div>
+  </div>
+
+  <!-- Footer -->
+  <div class="bg-orange soft-top">
+    <!-- App Icon -->
+    <div class="container soft-ends">
+      <div class="push-ends row soft-ends text-center">
+        <div class="col-sm-12">
+          <h1 class="section-header text-white flush-ends soft-top">
+            How To watch
+          </h1>
+          <h5 class="font-family-serif font-size-large text-white push-top">
+            Search “Crossroads” on your smart tv to get started.
+          </h5>
+          <div class="push-top" style="display: flex; justify-content: center;">
+            <ol class="font-family-serif font-size-large text-white text-left">
+              <li>Download the App</li>
+              <li>Set a time to watch</li>
+              <li>Get your family invovled</li>
+            </ol>
+          </div>
+          <div class="text-center push-top">
+            <img
+              style="margin: 0 60px; width: 100px !important; display: inline;"
```

Comment:
1. Lets try not to have inline styles unless it is absolutely necessary 
2. It is never a great idea to set width to an exact pixel (also not best practice to use an important tag unless it is absolutely necessary)